src/test/resources/recipes/MultipleDereferencesRecipes.java

```diff
@@ -54,7 +53,8 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                 public J visitMethodInvocation(J.MethodInvocation elem, ExecutionContext ctx) {
                     JavaTemplate.Matcher matcher;
                     if ((matcher = before.matcher(getCursor())).find()) {
-                        doAfterVisit(new ShortenFullyQualifiedTypeReferences().getVisitor());
+                        doAfterVisit(new org.openrewrite.java.ShortenFullyQualifiedTypeReferences().getVisitor());
+                        doAfterVisit(new org.openrewrite.staticanalysis.UnnecessaryParentheses().getVisitor());
```

Comment:
Actually, we have `UnnecessaryParenthesesVisitor` in `rewrite-java`: `org.openrewrite.java.cleanup.UnnecessaryParenthesesVisitor`.